src/components/Results/Results.css

```diff
@@ -1,10 +1,29 @@
+.results-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 1em;
+  margin: 1em;
+}
+
 /* --------- MEDIA QUERIES -----------  */
 
 @media screen and (min-width: 620px) {
+  .results-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
I think we can refactor these media queries - grid has a repeat/autofit/minmax feature, that will automatically update the number of cards per page based on the page size. This should DRY up our media queries significantly.

Reply:
Oh sweet! I haven't used that before